extensions/jaxp/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: "JAXP"
+metadata:
+  categories:
+  - "xml"
```

Comment:
I wonder if we should make it unlisted or not? It looks like a very technical one?

Reply:
+1 for making it unlisted

Reply:
I think that there may well exist end users needing DocumentBuilder and XPath, but I have no problem to unlist if you insist on that.

Reply:
unlisted in f3e6f14df24b908373f09a8defa29a424aaa0d0e